package.json

```diff
@@ -2,10 +2,41 @@
   "name": "@blockr/blockr-p2p-lib",
   "version": "1.0.0",
   "description": "Blockr P2P node library",
-  "main": "index.js",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "build": "tsc -p tsconfig.json",
+    "build-watch": "tsc -w -p tsconfig.json",
+    "test": "exit 0;",
+    "lint": "tslint --project tsconfig.json",
+    "dev": "npm run build && npm run lint"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ESD6-blockr/blockr-p2p-lib.git"
   },
   "author": "Blockr (Hugo van de Vliert, Ronald van den Broek)",
-  "license": "MIT"
+  "license": "MIT",
+  "dependencies": {
+    "@blockr/blockr-logger": "^1.1.0-3",
+    "@types/figlet": "^1.2.0",
+    "chalk": "^2.4.2",
```

Comment:
Remove unused package

Reply:
All packages to remove  

  "@types/figlet": "^1.2.0",
    "chalk": "^2.4.2",
    "figlet": "^1.2.1",
    "path": "^0.12.7",

Reply:
Remove dev dependancy:

    "@types/object-hash": "^1.2.0",